Mathlib/Algebra/Module/Presentation/Tensor.lean

```diff
@@ -13,18 +13,16 @@ Given presentations of two `A`-modules `M₁` and `M₂`, we obtain a presentati
 
 -/
 
-universe w w₁₀ w₁₁ w₂₀ w₂₁ u v₁ v₂
-
 namespace Module
 
 open TensorProduct
 
-variable {A : Type u} [CommRing A] {M₁ : Type v₁} {M₂ : Type v₂}
+variable {A : Type*} [CommRing A] {M₁ : Type*} {M₂ : Type*}
   [AddCommGroup M₁] [AddCommGroup M₂] [Module A M₁] [Module A M₂]
 
 namespace Relations
 
-variable (relations₁ : Relations.{w₁₀, w₁₁} A) (relations₂ : Relations.{w₂₀, w₂₁} A)
+variable (relations₁ : Relations A) (relations₂ : Relations A)
 
```

Comment:
And also here (and below).

Reply:
As for this one, would you like to continue in the discussion on Zulip?
[#mathlib4 > Task 26: Replace Type u by Type&#42; wherever possible @ 💬](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Task.2026.3A.20Replace.20Type.20u.20by.20Type*.20wherever.20possible/near/513659767)